src/components/Header/Header.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+
+function Header() {
+  const [dropdown, setDropdown] = useState(false);
+  return (
+    <header>
+      <nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
+        <a class="navbar-brand" href="#">
+          Navbar
+        </a>
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-toggle="collapse"
+          data-target="#navbarsExampleDefault"
+          aria-controls="navbarsExampleDefault"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="collapse navbar-collapse" id="navbarsExampleDefault">
+          <ul class="navbar-nav mr-auto">
+            <li class="nav-item active">
+              <a class="nav-link" href="#">
+                Home <span class="sr-only">(current)</span>
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">
+                Link
+              </a>
+            </li>
+            <li class="nav-item">
+              <a
+                class="nav-link disabled"
+                href="#"
+                tabindex="-1"
+                aria-disabled="true"
+              >
+                Disabled
+              </a>
+            </li>
+            <li
+              onChange={() => setDropdown(dropdown === true)}
+              class="nav-item dropdown"
+            >
+              <a
+                class="nav-link dropdown-toggle"
+                href="#"
+                id="dropdown01"
+                role="button"
+                data-toggle={dropdown}
+                aria-haspopup="true"
+                aria-expanded="false"
+              >
+                Dropdown
+              </a>
+              {dropdown ? (
+                <div class="dropdown-menu" aria-labelledby="dropdown01">
```

Comment:
bootstrap uses a show class on this div that will organically show/hide the drop down. I would do something like className={"dropdown-menu " + (dropdown ? 'show' : '') } instead of hiding and showing the entire div